main.py

```diff
@@ -356,6 +362,53 @@ def __init__(self, *args):
 		self.actionExit.triggered.connect(self.onExitClicked)
 		self.actionSettings.triggered.connect(self.onSettingsClicked)
 
+		if not "search_for_updates" in self.settings.settings or not "buildId" in self.settings.settings:
+			self.settings.settings["search_for_updates"] = False
+			self.settings.settings["buildId"] = ""
+			self.settings.save()
```

Comment:
There's better ways to handle settings (Python has getters with defaults, for example). I think we should fix this globally (like using getters / setters to access settings, which automatically handle defaults).

*Not really an issue with your PR, just a reminder because it keeps coming up (leading to hacks like this)*